op-mode-definitions/restart-frr.xml.in

```diff
@@ -52,28 +46,46 @@
       </leafNode>
       <leafNode name="ripng">
         <properties>
-          <help>Restart Routing Information Protocol NG (RIPng) routing daemon</help>
+          <help>Restart IPv6 Routing Information Protocol (RIPng) routing daemon</help>
         </properties>
         <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon ripngd</command>
       </leafNode>
-      <leafNode name="static">
+      <leafNode name="isis">
         <properties>
-          <help>Restart static routing daemon</help>
+          <help>Restart Intermediate System to Intermediate System (IS-IS) routing daemon</help>
         </properties>
-        <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon staticd</command>
+        <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon isisd</command>
       </leafNode>
-      <leafNode name="zebra">
+      <leafNode name="pim6">
         <properties>
-          <help>Restart Routing Information Base (RIB) manager daemon</help>
+          <help>Restart IPv6 Protocol Independent Multicast (PIM) daemon</help>
         </properties>
-        <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon zebra</command>
+        <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon pim6d</command>
+      </leafNode>
+      <leafNode name="ldp">
+        <properties>
+          <help>Restart Label Distribution Protocol (LDP) daemon used by MPLS</help>
+        </properties>
+        <command>sudo ${vyos_op_scripts_dir}/restart_frr.py --action restart --daemon ldpd</command>
+      </leafNode>
+      <leafNode name="eigrp">
```

Comment:
We don't have `eigrp`

Reply:
Cant this remain enabled due to the concurrent work of https://vyos.dev/T2472 ?

Please let me know if I should disable it anyway.

Reply:
We can't implement `eigrp` it cannot generate config correctly due to FRR bugs.
For example:
```
r4(config)# router eigrp 65001
r4(config-router)# redistribute connected
% Configuration failed.

Error type: validation
r4(config-router)#
r4(config-router)# redistribute static
% Configuration failed.

Error type: validation
r4(config-router)#

r4# show version
FRRouting 9.0.1 (r4) on Linux(6.1.53-amd64-vyos).
```
bug with passive interface
```
r4(config-router)# passive-interface eth0
% Configuration failed.

Error type: validation
r4(config-router)#
```
And so on..